docs/program/extend/modular-resources/examples/add-rplidar-module.md

```diff
@@ -4,6 +4,8 @@ linkTitle: "Add a Rplidar as a Modular Component"
 weight: 40
 type: "docs"
 description: "How to add a Rplidar as a modular component of your robot."
+image: "/program/img/modular-resources/rplidar-on-robot.png"
+imageAlt: "A rplidar mounted to a Viam rover."
```

Comment:
[nit] Is it prounced "R-P-LIDAR?" If so, then it should be "An RPLidar" not "a" I think. If "replydar" is how it's said or something, then never mind.

Reply:
Oh true 😆